CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # CHANGELOG
 
+## Version 3.7.0 GA (01/22/2025)
+
+### Enhancements
+
+* Transition to azure-monitor-opentelemetry-autoconfigure 1.0.0-beta.1, in place of azure-monitor-opentelemetry-exporter [#4037](https://github.com/microsoft/ApplicationInsights-Java/pull/4037)
```

Comment:
I think we can leave this out since it's not a user-facing change for Java agent users
```suggestion
```